src/ops.rs

```diff
@@ -147,6 +160,19 @@ impl<A: BitSetLike, B: BitSetLike> BitSetLike for BitSetOr<A, B> {
     }
 }
 
+impl<A: DrainableBitSet, B: DrainableBitSet> DrainableBitSet for BitSetOr<A, B> {
+    #[inline]
+    fn remove(&mut self, i: Index) -> bool {
+        if self.contains(i) {
+            self.0.remove(i);
```

Comment:
Can we actually do that? We don't know at this point if both 0 and 1 contain our element "i"

Reply:
Yes? If there is zero, removal doesn't do anything to it. I am not sure what could go wrong.